src/arcaflow_plugin_sdk/test_plugin.py

```diff
@@ -52,5 +52,25 @@ def cleanup():
         self.assertEqual("Hello world!\n", e.getvalue())
 
 
+@plugin.step(
+    "incorrect-return",
+    "Incorrect Return",
+    "A test that doesn't include the output ID.",
+    {"success": EmptyTestOutput},
+)
+def incorrect_return_step(
+    _: EmptyTestInput,
+):  # Skip return type, since we're purposefully not doing it right.
+    return EmptyTestOutput()
+
+
+class CallStepTest(unittest.TestCase):
+    def test_incorrect_return_args_count(self):
+        s = plugin.build_schema(incorrect_return_step)
+
+        with self.assertRaises(schema.BadArgumentException):
+            s.call_step(self.id(), "incorrect-return", EmptyTestInput())
```

Comment:
Instead of `EmptyTestInput()`, consider passing `None` or `{}` or `object()`, here.

Reply:
Given that the definition of `EmptyTestInput` is useless and should be removed (and replaced with `Any` in the function signatures which reference it), we'll presumably still want some value to pass in here.  Since it will be ignored by the function anyway, we want some sort of "null" value, and, in Python, those are the obvious choices.

But, since you [don't seem to be interested](https://github.com/arcalot/arcaflow-plugin-sdk-python/pull/134#discussion_r1651458691) in getting rid of `EmptyTestInput` (even though you thought it was important enough to rename it), I'm going to resolve this conversation, too.